File: 2076-sum-of-digits-of-string-after-convert/2076-sum-of-digits-of-string-after-convert.py
class Solution:
    def getLucky(self, s: str, k: int) -> int:
        nustr=""
        for ch in s:
            nustr=nustr+str(ord(ch)-ord('a')+1)
        while(k>0):
            sumy=0
            for cha in nustr:
                sumy=sumy+int(cha)
            nustr=str(sumy)
            k=k-1
        return int(nustr)            
        

File: 2076-sum-of-digits-of-string-after-convert/2076-sum-of-digits-of-string-after-convert.java
class Solution {
    public int getLucky(String s, int k) {
        StringBuilder sb=new StringBuilder();
        for(int i=0;i<s.length();i++){
            sb.append(s.charAt(i)-'a'+1);
        }
        String num=sb.toString();
        int ans=0;
        while(k-->0){
            int sum=0;
            for(char ch:num.toCharArray()){
                sum=sum+ch-'0';

            }
            num=String.valueOf(sum);

        }
        return Integer.parseInt(num);
        
    }
}